### ISSUE TITLE
Is it possible to send test / simulation orders?

### ISSUE THREAD

**Issue by ThallyssonKlein** [jaggedsoft/node-binance-api#674]
I would like to know if it is possible to test the sending of orders simulated, without using real money initially.

**Comment by emansammut** [comment]
Yes you can by setting up an apikey and secret key on https://testnet.binance.vision/
But please note that the liquidity is very low on the testnet in which it wouldn't be as realastic but to setup the config to work with testnet, you would need to point your trades to the testnet using this package by setting up the config similar to below:
```
const binance = new Binance().options({
            APIKEY: '',
            APISECRET: '',
            adjustForTimeDifference: true,
            urls: {
                base: "https://testnet.binance.vision/api/"
            }
});
```

**Comment by ThallyssonKlein** [comment]
Why does the return of orders become an empty object using testnet?

**Comment by emansammut** [comment]
if you're using test: true in the config it will return an empty object, however if you use testnet account without test:true, it will try to fill the order but as mentioned liquidity on testnet is very low and will give a false perception to reality. I would first try without test:true, to make sure you get the response that you you need and handle accordingly in case of any errors. After that use test:true, to simulate orders as expected and try to remove any further errors regarding minNotional, qty filters etc. That's how I progressed before moving to the live account.

**Comment by gurachan** [comment]
> Yes you can by setting up an apikey and secret key on https://testnet.binance.vision/
> But please note that the liquidity is very low on the testnet in which it wouldn't be as realastic but to setup the config to work with testnet, you would need to point your trades to the testnet using this package by setting up the config similar to below:
> 
> ```
> const binance = new Binance().options({
>             APIKEY: '',
>             APISECRET: '',
>             adjustForTimeDifference: true,
>             urls: {
>                 base: "https://testnet.binance.vision/api/"
>             }
> });
> ```

I followed this and the candles data from ethusdt@kline_15m is so different xD and it looks like a bugged result.

 

i have mine setup like this 

```
const binance = new Binance().options({
    APIKEY: Account.APIKEY,
    APISECRET: Account.APISECRET,
})
```
..

then it will bug if i try to do this
```
const binance = new Binance().options({
    APIKEY: Account.TESTKEY,
    APISECRET: Account.TESTSECRET,
    adjustForTimeDifference: true,
    urls: {
         base: "https://testnet.binance.vision/api/"
    }
})
```

even if i overdo it like this 

```
const binance = new Binance().options({
    APIKEY: Account.TESTKEY,
    APISECRET: Account.TESTSECRET,
    adjustForTimeDifference: true,
    urls: {
         base: "https://testnet.binance.vision/api/",
         combineStream: 'wss://testnet.binance.vision/stream?streams=',
         stream: 'wss://testnet.binance.vision/ws/'
    }
})
```
 
 
 they will endup like this 
 
![image](https://user-images.githubusercontent.com/12135498/119550128-96718e80-bdca-11eb-9f64-716ff6c6bfd4.png)

the expected result is 

![image](https://user-images.githubusercontent.com/12135498/119550173-aa1cf500-bdca-11eb-89fe-62dc9f659fae.png)

**Comment by kanish-111** [comment]
even i want to place a order in this, but it shows a error when using testnet
i used this
   let base = 'https://testnet.binance.vision/api/';
    let wapi = 'https://testnet.binance.vision/wapi/';
    let sapi = 'https://testnet.binance.visionsapi/';
